### ISSUE TITLE
generate_skeleton has no results

### ISSUE THREAD

**Issue by xilai0715** [VAST-AI-Research/UniRig#22]
Thanks for your sharing.

**I can`t get the generate_skeleton  results.**

cmd:
$ bash launch/inference/generate_skeleton.sh --input examples/bird.glb --output results/bird_skeleton.fbx
  0%|                                                                                                                                                                                          | 0/1 [00:00<?, ?it/s]Info: Deleted 2 data-block(s)
Now processing examples/bird.glb...
Now processing examples/bird.glb...
11:29:22 | INFO: Data are loaded, start creating Blender stuff
11:29:22 | INFO: Blender create Mesh node SK_tucano_bird.001
11:29:22 | INFO: glTF import finished in 0.02s
save to: tmp\examples/bird
100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 1/1 [00:00<00:00,  2.53it/s] 
1 models processed
done
D:\fxq\soft\anconda\envs\UniRig\Lib\site-packages\timm\models\layers\__init__.py:48: FutureWarning: Importing from timm.models.layers is deprecated, please import via timm.layers
  warnings.warn(f"Importing from {__name__} is deprecated, please import via timm.layers", FutureWarning)
Seed set to 12345
load task config: configs/task/quick_inference_skeleton_articulationxl_ar_256.yaml
load data config: configs/data\quick_inference.yaml
load transform config: configs/transform\inference_ar_transform.yaml
load tokenizer config: configs/tokenizer\tokenizer_parts_articulationxl_256.yaml
load model config: configs/model\unirig_ar_350m_1024_81920_float32.yaml
D:\fxq\soft\anconda\envs\UniRig\Lib\site-packages\huggingface_hub\file_download.py:1150: FutureWarning: `resume_download` is deprecated and will be removed in version 1.0.0. Downloads always resume when possible. If you want to force a new download, use `force_download=True`.
  warnings.warn(
load system config: configs/system\ar_inference_articulationxl.yaml
Using bfloat16 Automatic Mixed Precision (AMP)
Using default `ModelCheckpoint`. Consider installing `litmodels` package to enable `LitModelCheckpoint` for automatic upload to the Lightning model registry.
GPU available: True (cuda), used: True
TPU available: False, using: 0 TPU cores
HPU available: False, using: 0 HPUs
D:\fxq\soft\anconda\envs\UniRig\Lib\site-packages\lightning\pytorch\trainer\connectors\logger_connector\logger_connector.py:76: Starting from v1.9.0, `tensorboardX` has been removed as a dependency of the `lightning.pytorch` package, due to potential conflicts with other packages in the ML ecosystem. For this reason, `logger=True` will use `CSVLogger` as the default logger, unless the `tensorboard` or `tensorboardX` packages are found. Please `pip install lightning[extra]` or one of them to enable TensorBoard support by default
Restoring states from the checkpoint path at C:\Users\cd_fengxq\.cache\huggingface\hub\models--VAST-AI--UniRig\snapshots\e53ad6237db5cea261727c5d50b67d140a4b9571\skeleton\articulation-xl_quantization_256\model.ckptLOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]
Loaded model weights from the checkpoint at C:\Users\cd_fengxq\.cache\huggingface\hub\models--VAST-AI--UniRig\snapshots\e53ad6237db5cea261727c5d50b67d140a4b9571\skeleton\articulation-xl_quantization_256\model.ckpt
D:\fxq\soft\anconda\envs\UniRig\Lib\site-packages\lightning\pytorch\trainer\connectors\data_connector.py:425: The 'predict_dataloader' does not have many workers which may be a bottleneck. Consider increasing the value of the `num_workers` argument` to `num_workers=15` in the `DataLoader` to improve performance.
Predicting DataLoader 0:   0%|                                                                                                                                                                 | 0/1 [00:00<?, ?it/s]D:\fxq\soft\anconda\envs\UniRig\Lib\contextlib.py:105: FutureWarning: `torch.backends.cuda.sdp_kernel()` is deprecated. In the future, this context manager will be removed. Please see `torch.nn.attention.sdpa_kernel()` for the new context manager, with updated signature.
  self.gen = func(*args, **kwds)
If inputs_embeds is passed as model-specific keyword input then model has to be an encoder-decoder and not a OPTForCausalLM.
Predicting DataLoader 0: 100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 1/1 [00:00<00:00,  2.23it/s]
done

**But,I can get generate_skin.sh results.**

FBX export starting... 'results/bird_skin.fbx'
(bpy.data.armatures['Armature'], 'POSE')
export finished in 0.0310 sec.
Predicting DataLoader 0: 100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 1/1 [00:04<00:00,  0.25it/s]
done

![Image](https://github.com/user-attachments/assets/b2c8c448-d224-47a7-b755-2a0d30aeadd1)

**Comment by xilai0715** [comment]
I solved this.
The reason for this error is that the version of the transfomer is incorrect。
should be：transformers-4.51.3.
btw,you can print the error info  in **src\system\ar.py**
like this:
![Image](https://github.com/user-attachments/assets/25fa29f9-b8ab-4702-bf9d-d8524f40f930)
